### PR TITLE
Abbreviations support

### LINKED ISSUE #84: [FR] Add site abbreviation support
Ether a setting in the config for site abbreviations i.e. (abbreviations=["abr":"A Bare Rump", ....]  )
or as a separate file for them

As discussed in discord https://discord.com/channels/957516730727563285/957516731654488091/1015037937206956155

### PR DESCRIPTION
**Pull request by DirtyRacer1337**
Resolves #84

### CODE REVIEW

File: namer/database.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    'ihaw': 'IHaveAWife',
+    'iktg': 'IKnowThatGirl',
+    'il': 'ImmoralLive',
+    'kha': 'KarupsHA',
+    'kow': 'KarupsOW',
+    'kpc': 'KarupsPC',
+    'la': 'LatinAdultery',
+    'lcd': 'LittleCaprice',
+    'lhf': 'LoveHerFeet',
+    'littlecapricedreams': 'Little Caprice Dreams',
```
Comment (4c0d3r) [inline namer/database.py:101]: shouldn't be needed, namer should handle this.

File: namer/database.py [hunk 2]
```diff
@@ -0,0 +1,299 @@
+abbreviations = {
```
Comment (4c0d3r) [inline namer/database.py:1]: my plan was to make it configurable via a map in the NamerConfig, ['aa': 'AmateurAllure]

This file would be a good seed for the config, but ultimately I have no idea what some site ripper might do, and no desire to build a list of everyone's different/conflicting fever dreams of how abbreviation should work.

**Comment by DirtyRacer1337** [comment]
It's just init version, this db from tpdb, so it should work the same

**Comment by 4c0d3r** [comment]
merged with additions to allow users to override the default set.

**Pull request closed without merging**